packages/connect/src/data/deviceAuthenticityConfig.ts

```diff
@@ -53,6 +53,20 @@ export const deviceAuthenticityConfig: DeviceAuthenticityConfig = {
             ],
         },
     },
+    T3B1: {
```

Comment:
manual change? there is s script preparing this file in nightly I think 🤔

Reply:
Yes, the script works, I just reordered it to keep it in the same order as in the rest of the codebase.

Reply:
just that the script might revert this change next time it runs maybe

Reply:
I checked, it shouldn't. It selects a key and rewrites the value. https://github.com/trezor/trezor-suite/blob/58da390d6157d7ee13d5ae8e773fe61a01582ab1/scripts/ci/check-connect-data.ts#L52